packages/cli/package.json

```diff
@@ -9,28 +9,43 @@
   "author": "Budibase",
   "license": "GPL-3.0",
   "scripts": {
-    "build": "pkg . --out-path build"
+    "prebuild": "rm -rf prebuilds 2> /dev/null && cp -r node_modules/leveldown/prebuilds prebuilds",
+    "build": "yarn prebuild && renamer --find .node --replace .fake 'prebuilds/**' && pkg . --out-path build && yarn postbuild",
+    "postbuild": "rm -rf prebuilds 2> /dev/null"
   },
   "pkg": {
     "targets": [
-      "node14-linux",
-      "node14-win",
-      "node14-macos"
+      "node16-linux",
+      "node16-win",
+      "node16-macos"
+    ],
+    "assets": [
+      "node_modules/@budibase/backend-core/dist/**/*",
+      "prebuilds/**/*"
     ],
     "outputPath": "build"
   },
   "dependencies": {
+    "@budibase/backend-core": "^1.0.212-alpha.7",
     "axios": "^0.21.1",
     "chalk": "^4.1.0",
+    "cli-progress": "^3.11.2",
```

Comment:
Given that we've pinned dependencies now in server, worker and backend-core, it could be worth pinning here too. it just solves so many problems around knowing exactly which versions are in use and keeping versions in sync across packages

Reply:
We do, but I've noticed cases where the same non-pinned version diverges across the lockfiles of backend packages. e.g. server and worker using different versions of koa middlewares. Pinning also adds some security improvements. A big example where this was useful was when we needed to purge the lockfiles for a CI problem when backend-core went to TS